src/i18n.js

```diff
@@ -18,11 +18,11 @@ i18n
       backendOptions: [
         { // LocalStorageBackend
           defaultVersion: 'v1',
-          expirationTime: (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') ? 1 : 7 * 24 * 60 * 60 * 1000
+          expirationTime: (!import.meta.env.NODE_ENV || import.meta.env.NODE_ENV === 'development') ? 1 : 7 * 24 * 60 * 60 * 1000
         },
         { // HttpBackend
           // ensure a relative path is used to look up the locales, so it works when loaded from /ipfs/<cid>
-          loadPath: 'locales/{{lng}}/{{ns}}.json'
+          loadPath: './locales/{{lng}}/{{ns}}.json'
```

Comment:
I saw some locale fetching errors and attempted debugging before I realized that this repo only has "explore.json" files.

```suggestion
          loadPath: 'locales/{{lng}}/{{ns}}.json'
```